executor/executor_simple.go

```diff
@@ -68,6 +68,8 @@ func (e *SimpleExec) Next() (*Row, error) {
 	switch x := e.Statement.(type) {
 	case *ast.UseStmt:
 		err = e.executeUse(x)
+	case *ast.FlushTableStmt:
+		err = e.executeFlushTable(x)
```

Comment:
no general `FlushStmt` in this PR yet, just add `FlushTableStmt`, which is a special case of the FLUSH syntax.